src/main/kotlin/no/nav/syfo/domain/Vurdering.kt

```diff
@@ -19,6 +19,8 @@ sealed class Vurdering(
     open val publishedAt: OffsetDateTime?,
 ) {
     abstract val journalpostType: JournalpostType
+    abstract val brevkode: BrevkodeType
+    abstract val dokumentTittel: String
```

Comment:
Mulig det var det du tenkte med punkt nr 2 i PR-beskrivelsen, men det kunne vel også vært en `abstract val journalforing: Journalforing` (evt `JournalforingRequest` / `JournalforingInfo` etc) sånn at man pakket inn alt med journalføring inn i én property i stedet for at `brevkode` ligger rett på Vurderingen?